caramel/config.py

```diff
@@ -5,7 +5,14 @@
 
 import argparse
 import logging
+from logging.config import dictConfig
 import os
+from pyramid.scripting import prepare
+from pyramid.paster import (
```

Comment:
It might read cleaner if you simply do "import pyramid.paster as paster"  and then do "paster.get_appsettings"  for those cases that need it.